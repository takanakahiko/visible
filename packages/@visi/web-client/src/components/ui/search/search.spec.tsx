import { render } from '@testing-library/react';
import React from 'react';

import { Search } from '.';

describe('Search', () => {
  it('matches snapshot', () => {
    const { container } = render(<Search id="foo">Search</Search>);
    expect(container.firstChild).toMatchSnapshot();
  });
});
