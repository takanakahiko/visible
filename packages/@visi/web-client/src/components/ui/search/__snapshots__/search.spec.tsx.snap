// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search matches snapshot 1`] = `
<form
  class="inline-flex"
  role="search"
>
  <input
    class="border-gray-400 border placeholder-gray-600 text-sm w-64 focus:outline-none focus:shadow-outline focus:border-primary-500 focus:bg-white disabled:opacity-75 disabled:cursor-not-allowed rounded py-2 px-4 rounded-full border-r-0 rounded-tr-none rounded-br-none"
    id="foo"
    value=""
  />
  <button
    class="rounded-full px-4 py-2 text-sm flex transition-colors duration-100 ease-in focus:outline-none focus:shadow-outline disabled:cursor-not-allowed disabled:opacity-75 bg-primary-500 text-white hover:bg-primary-600 active:bg-primary-700 border-l-0 rounded-tl-none rounded-bl-none"
    rel="noreferrer"
    target="_blank"
  >
    <span>
      Search
    </span>
  </button>
</form>
`;
